### ISSUE TITLE
Question about why the performance is different from the paper reporting

### ISSUE THREAD

**Issue by mikelmh025** [omihub777/ViT-CIFAR#6]
Nice implementation! Also my first time using PyTorch-lighting, it pretty nice!

I wonder why the performance is different from the paper's reporting -- 99% accuracy? Is it due to the fact that they have pre-trained model, or something else?

**Comment by sndnyang** [comment]
you are right.  Because in original ViT,  they use 300M images  +  1.2M imagenet images to train the model, then evaluate on all kinds of datasets.

**Comment by omihub777** [comment]
Thank you for your interest to my work!
As @sndnyang said, they pretrain ViT on huge image dataset in their paper. (I train ViT from scratch.)
